### DISCUSSION TITLE
微软晓晓语音没有撒娇选项

### DISCUSSION THREAD

**Discussion by linmister** [jing332/tts-server-android#97] [General]
大佬，我用微软的azure创建了密钥，但是选择风格的时候发现没有撒娇的选项，但是我看到微软的语音库里是有的

**Comment by jing332** [comment]
> 大佬，我用微软的azure创建了密钥，但是选择风格的时候发现没有撒娇的选项，但是我看到微软的语音库里是有的

Azure的有些地区可能不支持，你可以尝试清空APP缓存来重新请求语音数据列表。
